docs/user/using-an-OIDC-provider.md

```diff
@@ -108,6 +116,13 @@ helm install kubeapps bitnami/kubeapps \
   --set authProxy.cookieSecret=$(echo "not-good-secret" | base64) \
   --set authProxy.additionalFlags="{--cookie-secure=false}"
 ```
+> If you are serving Kubeapps under a subpath (eg., "example.com/subpath") you also need to set the `authProxy.oauthLoginURI` and `authProxy.oauthLogoutURI` flags, as well as the additional flag `--proxy-prefix`. For instance:
```

Comment:
Perhaps just have a separate section for this new info in this same doc, "Serving Kubeapps from a subpath while supporting OIDC" and referring to it in all 3 places?

Reply:
Yep, I think it's the best option for avoiding duplication. The point is: currently, changing the path is not supported in the chart values, so you have to manually patch the ingress object.

If we move these instructions to a separated section, perhaps we are giving much more importance to something not supported out-of-the-box by Kubeapps and wherefore I just decided to put it as a side note.

What do you think? Should we: 
i) support changing ingress path in the values.yaml or not?
ii) move these side notes to a separate section regardless of i) ?

Reply:
Update: after a conversation with @andresmgot , it seems it is possible to change the path by using the [Values.ingress.extraHosts](https://github.com/kubeapps/kubeapps/blob/master/chart/kubeapps/templates/ingress.yaml#L33). I'll check it just to confirm and will update the docs accordingly.